var/spack/repos/builtin/packages/slepc/package.py

```diff
@@ -38,34 +41,51 @@ class Slepc(Package):
 
     variant('arpack', default=True, description='Enables Arpack wrappers')
     variant('blopex', default=False, description='Enables BLOPEX wrappers')
+    variant('double',  default=True,
+            description='Switches between single and double precision')
+    variant('complex', default=False, description='Build with complex numbers')
+    variant('int64', default=False,
+            description='Compile with 64bit indices')
```

Comment:
Is the only purpose of these variants to control the petsc build? If that's the case, they can be removed, as it's already possible to control the petsc build from the spec like so:
```console
$ spack install slepc ^petsc+double
```